dace/transformation/dataflow/redundant_array.py

```diff
@@ -97,4 +97,107 @@ def gnode(nname):
             RedundantArray._arrays_removed += 1
 
 
+class RedundantArrayInverse(pm.Transformation):
```

Comment:
Can we change the name? The `Inverse` implies that there is some kind of inversion operation. What about `RedundantSecondArray`?

Reply:
Fixed